doc/UseCases/uc-ss_x-road_use_case_model_for_security_server_management_1.4_Y-883-4.md

```diff
@@ -0,0 +1,2709 @@
+
+**System**: Security server
+
+**Level**: User task
+
+**Component:** Security server
+
+**Actors:** SS administrator
+
+**Brief Description**: SS administrator exports the internal TLS
+certificate of the security server to the local file system.
+
+**Preconditions**: An internal TLS certificate has been created.
+
+**Postconditions**: The internal TLS certificate has been exported.
+
+**Trigger**: SS administrator wants to export an internal TLS
+certificate.
+
+**Main Success Scenario**:
+
+1.  SS administrator selects to export the internal TLS certificate of
+    the security server.
+
+2.  System prompts a tar.qz file for downloading, that contains the TLS
+    certificate in PEM and CER format.
+
+3.  SS administrator saves the file to the local file system.
+
+**Extensions**: -
+
+**Related information**: -
+
+### 3.14 UC SS\_13: View the List of Configuration Backup Files
+
+**System**: Security server
+
+**Level**: User task
+
+**Component:** Security server
+
+**Actors:** SS administrator
+
+**Brief Description**: SS administrator views the list of configuration
+backup files saved in the system configuration.
+
+**Preconditions**: *-*
+
+**Postconditions**: The list of configuration backup files has been
+displayed to SS administrator.
+
+**Trigger**: SS administrator wants to view the list of configuration
+backup files.
+
+**Main Success Scenario**:
+
+1.  SS administrator selects to view the list of configuration backup
+    files.
+
+2.  System displays the list of backup files. For each file, the
+    following information is displayed:
+
+    -   the file name of the backup file.
+
+    The SS administrator has a possibility to choose amongst the following actions:
+
+    -   back up configuration: 3.15;
+    
+    -   upload a backup file: 3.19;
+    
+    -   download a backup file: 3.17;
+    
+    -   restore system configuration from a backup file: 3.16;
+    
+    -   delete a backup file: 3.18.
+
+**Extensions**: -
+
+**Related information**: -
+
+### 3.15 UC SS\_14: Back Up Configuration
+
+**System**: Security server
+
+**Level**: User task
+
+**Component:** Security server
+
+**Actors:** SS administrator
+
+**Brief Description**: SS administrator backs up the security server
+configuration.
+
+**Preconditions**: -
+
+**Postconditions**: An audit log record for the event is created.
+
+**Trigger**: SS administrator wants to back up the security server
+configuration.
+
+**Main Success Scenario**:
+
+1.  SS administrator selects to back up the security server
+    configuration.
+
+2.  System runs the backup script, that
+
+    a.  creates a dump file of the database (including the schema) to
+        the location /var/lib/xroad/dbdump.dat, that contains the
+        contents of the security server database;
+
+    b.  creates the backup file containing the database dump file and
+        the following directories:
+
+        -   /etc/xroad/
```

Comment:
Block with gray background could be fixed here and later in this document.

Reply:
fixed